docs/_data/portland-2021-config.yaml

```diff
@@ -161,8 +161,7 @@ cfp:
   video_by: "March 31, 2021"        # 3 weeks before the conf
 
 unconf:
-  url: http://bit.ly/wtd-schedule-pdx-2021
-
+  url: https://docs.google.com/spreadsheets/d/1bE3_p0EkfPrcQzfbPj-7B0mGZQwzOo2V10jEqiVQ6hk/edit#gid=0
```

Comment:
Is this right? We are replacing the read only version here with the read write link. We try to send people to the read only version when possible, because the read write version only allows 100 viewers at the same time.

Reply:
This also applies to other links to this document in this PR.

Reply:
Oops, that might be my mistake. @canncrochet 

@mxsasha did we have two config variables for it?

Reply:
Looks we need to revert the config `http://bit.ly/wtd-schedule-pdx-2021` but keep the wording changes, right?

Reply:
Yes 👍